ibis/backends/dask/execution/generic.py

```diff
@@ -401,6 +403,27 @@ def wrap_case_result(raw: np.ndarray, expr: ir.ValueExpr):
     return result
 
 
+@execute_node.register(ops.SearchedCase, list, list, object)
+def execute_searched_case_dask(op, whens, thens, otherwise, **kwargs):
```

Comment:
This looks fine to me but is this related to the no-op windows change? Not super clear to me.

Reply:
https://github.com/ibis-project/ibis/pull/2843#discussion_r668123946